### PR TITLE
Clean up spec files

### PR DESCRIPTION
**Pull request by xsuchy**
See individual commits for reasoning.

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

**Comment by mhabrnal** [comment]
add to whitelist

**Comment by mhabrnal** [comment]
test this please

File: libreport.spec.in [hunk 1]
```diff
@@ -320,7 +320,7 @@ CFLAGS="%{optflags} -Werror" %configure \
 %if %{without bugzilla}
         --without-bugzilla \
 %endif
-%if 0%{?rhel:%{rhel} < 7}%{?fedora:%{fedora} < 18}
+%if 0%{?rhel} < 7
```
Comment (jfilak) [inline libreport.spec.in]: On Fedora, this condition equals `"if 0 < 7"` which is of cores true, but we don't want to pass the arguments below to the ./configure script on Fedora.

**Comment by xsuchy** [comment]
Updated.

**Comment by jfilak** [comment]
Thank you!

**Pull request merged**